CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# dbt_salesforce 0.7.0
+🎉 Salesforce Package Updates 🎉
+
+We are updating the Salesforce package! To improve its utility, the changes include the following:
+## Features
```

Comment:
Can you also add any new variables that you updated the README.

Reply:
I would also include any new variables that were added. If you want to just link to the section of the README that details these variables then I think that would be good as well.